wagtail/admin/views/pages.py

```diff
@@ -754,6 +754,13 @@ def move_confirm(request, page_to_move_id, destination_id):
     if not page_to_move.permissions_for_user(request.user).can_move_to(destination):
         raise PermissionDenied
 
+    if not Page._slug_is_available(page_to_move.slug, destination, page=page_to_move):
+        messages.error(
+            request,
+            _("The slug '{0}' is already in use at the selected parent page.".format(page_to_move.slug))
```

Comment:
Add text about what to do to be able to move the page. Eg: "Make the slug unique and try again."